src/Umbraco.Web.UI/Umbraco.Web.UI.csproj

```diff
@@ -9,6 +9,8 @@
   <Import Project="..\Umbraco.Cms.Targets\buildTransitive\Umbraco.Cms.Targets.targets" />
   <ItemGroup>
     <ProjectReference Include="..\Umbraco.Cms\Umbraco.Cms.csproj" />
+    <!-- Add design/build time support for EF Core migrations -->
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="8.0.0-rc.2.*" PrivateAssets="all" />
```

Comment:
I assume this reference is added to support the `dotnet ef migrations` commands that are mentioned on the `UmbracoDbContext`:
https://github.com/umbraco/Umbraco-CMS/blob/d5ff80352e2acbeec5ff9cf3de4a0a8d7be79a8e/src/Umbraco.Cms.Persistence.EFCore/UmbracoDbContext.cs#L14-L30

But I couldn't get these commands to work, so I'm not really sure 😇

Reply:
Thats the reason yes, and it should work, but otherwise it should not be related to this